src/parameters/create_parameters.jl

```diff
@@ -80,13 +79,13 @@ function create_parameter_set(config::AtmosConfig)
             rrtmgp_params,
             insolation_params,
             microphysics_params = microphys_params,
-            surfacefluxes_params = surf_flux_params,
+            surface_fluxes_params = surf_flux_params,
             turbconv_params,
-            sponge_params,
         ),
     )
-    # TODO: Add parameter logging option from config
-    # logfilepath = joinpath(@__DIR__, "logfilepath_$FT.toml")
-    # CP.log_parameter_information(toml_dict, logfilepath)
+    if parsed_args["log_params"]
+        logfilepath = joinpath(@__DIR__, "$(parsed_args["job_id"])_$FT.toml")
+        CP.log_parameter_information(toml_dict, logfilepath)
+    end
```

Comment:
From the help of `@__DIR__`: `  Expand to a string with the absolute path to the directory of the file containing the macrocall`, so I think that writes in the folder where this file particular lives, which might be something like `.julia/packages/ClimaAtmos/packages/XASy1234/src/`. 

Also, it would be really amazing if we didn't access `parsed_args` in this function, so that we can decouple the generation of the parameters with the parsed arguments.

Reply:
The function to log the parameters could be moved to the `AtmosConfig` constructor so that the only reference to `parsed_args` in this function is for the precipitation.

Reply:
Thanks for catching this! I can move it. I will try to get rid of the tau precip override next week, so if everything goes well the `parsed_args` will be separated out.